src/main.rs

```diff
@@ -725,192 +716,175 @@ impl Fastmod {
-            .short("g")
-            .long("glob")
-            .value_name("GLOB")
-            .multiple(true)
-            .conflicts_with("iglob")
-            .help("A space-delimited list of globs to process.")
-        )
-        .arg(
-            Arg::with_name("hidden")
-                .long("hidden")
-                .help("Search hidden files.")
-        )
-        .arg(
-            Arg::with_name("iglob")
-            .long("iglob")
-            .value_name("IGLOB")
-            .multiple(true)
-            .help("A space-delimited list of case-insensitive globs to process.")
-        )
-        .arg(
-            Arg::with_name("accept_all")
-                .long("accept-all")
-                .help("Automatically accept all changes (use with caution)."),
-        )
-        .arg(
-            Arg::with_name("print_changed_files")
-                .long("print-changed-files")
-                .help("Print the paths of changed files. (Recommended to be combined with --accept-all.)"),
-        )
-        .arg(
-            Arg::with_name("fixed_strings")
-                .long("fixed-strings")
-                .short("F")
-                .help("Treat REGEX as a literal string. Avoids the need to escape regex metacharacters (compare to ripgrep's option of the same name).")
-        )
-        .arg(
-            Arg::with_name("match")
-                .value_name("REGEX")
-                .help("Regular expression to match.")
-                .required(true)
-                .index(1),
-        )
-        .arg(
-            Arg::with_name("subst")
-             // TODO: support empty substitution to mean "open my
-             // editor at instances of this regex"?
-             .required(true)
-             .help("Substitution to replace with.")
-             .index(2),
-        )
-        .get_matches();
-    let multiline = matches.is_present("multiline");
+    let args = Args::parse();
+
     let dirs = {
-        let mut dirs: Vec<_> = matches
-            .values_of("dir")
-            .unwrap_or_default()
-            .chain(matches.values_of("file_or_dir").unwrap_or_default())
-            .collect();
+        let mut dirs: Vec<_> = args.dir.into_iter().chain(args.file_or_dir).collect();
         if dirs.is_empty() {
-            dirs.push(".");
+            dirs.push(PathBuf::from("."));
         }
         dirs
     };
-    let ignore_case = matches.is_present("ignore_case");
-    let file_set = get_file_set(&matches);
-    let accept_all = matches.is_present("accept_all");
-    let hidden = matches.is_present("hidden");
-    let print_changed_files = matches.is_present("print_changed_files");
-    let regex_str = matches.value_of("match").expect("match is required!");
-    let subst = matches.value_of("subst").expect("subst is required!");
-    let (maybe_escaped_regex, subst) = if matches.is_present("fixed_strings") {
-        (regex::escape(regex_str), subst.replace("$", "$$"))
+
+    let file_set = {
```

Comment:
In my opinion, inlining this function does not make the code better. Why do we need to do it?